src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/locations/LocationsService.kt

```diff
@@ -70,6 +70,11 @@ class LocationsService(
   suspend fun amendLocation(event: LocationDomainEvent) {
     doUpdateLocation(event, "amend") { nomisLocationId, location ->
       nomisApiService.updateLocation(nomisLocationId, toUpdateLocationRequest(location))
+      nomisApiService.updateLocationCapacity(nomisLocationId, UpdateCapacityRequest(location.capacity?.maxCapacity, location.capacity?.maxCapacity))
+      nomisApiService.updateLocationCertification(
+        nomisLocationId,
+        UpdateCertificationRequest(location.certification?.capacityOfCertifiedCell ?: 0, location.certification?.certified ?: false),
+      )
```

Comment:
I will change the Nomis updateLocation() endpoint to do all three as a separate change